systemd/system-generators/coreos-autologin-generator

```diff
@@ -12,6 +12,7 @@ fi
 
 # Use the first path provided, the "Normal" target for generators.
 if [[ $# -gt 0 ]]; then
+    TMPDIR="/run" # /tmp isn't a tmpfs yet when system-generators are run, reuse /run.
```

Comment:
Could you note what is actually using $TMPDIR? it isn't obvious from the script. And does it need to be exported?